llvm/lib/Target/AArch64/GISel/AArch64InstructionSelector.cpp

```diff
@@ -1994,7 +1994,108 @@ bool AArch64InstructionSelector::selectVectorAshrLshr(
 
 bool AArch64InstructionSelector::selectVaStartAAPCS(
     MachineInstr &I, MachineFunction &MF, MachineRegisterInfo &MRI) const {
-  return false;
+
+  if (MF.getSubtarget<AArch64Subtarget>().isCallingConvWin64(
```

Comment:
I assume the subtarget is already available as a member in AArch64InstructionSelector

Reply:
Replaced with `STI`